.eslintignore

```diff
@@ -1,3 +1,4 @@
-**/node_modules
+node_modules/*
+samples/node_modules/*
```

Comment:
`**/node_modules` *should* cover both of these, and any `node_modules` further down the hierarchy.